src/views/forms/AccountManagementForms/SourceSelectMenu.tsx

```diff
@@ -92,7 +90,8 @@ export const SourceSelectMenu = ({
           groupLabel: stringGetter({ key: STRING_KEYS.EXCHANGES }),
           items: exchangeItems,
         },
-        walletType !== WalletType.Privy &&
+        // only unblock privy wallets for withdrawals
+        type === TransferType.withdrawal &&
```

Comment:
@rosepuppy this is what you were talking about, right?

Reply:
actually so it should only block if it is deposit and privy, so something like:
`type === TransferType.deposit && walletType !== WalletType.Privy`

Reply:
omg 🤦 sorry i gotta slow down when making these. yeah this should have been really clear, my bad